zerver/tests/test_invite.py

```diff
@@ -2834,14 +2834,22 @@ def test_only_owner_can_change_create_multiuse_invite_group(self) -> None:
         self.login("iago")
         result = self.client_patch(
             "/json/realm",
-            {"create_multiuse_invite_group": orjson.dumps(full_members_system_group.id).decode()},
+            {
+                "create_multiuse_invite_group": orjson.dumps(
+                    {"new": full_members_system_group.id}
+                ).decode()
```

Comment:
I guess we can also add a test case when the setting is set to anonymous or user defined group, for the sake of completion.

Reply:
Yeah, it would be reasonable to just make one of the values in this test use that API, perhaps just adding `iago` individually too, without doing more API calls.